src/Workspaces/Directory.Build.targets

```
@@ -0,0 +1,7 @@
+<Project>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
+
+  <ItemGroup>
+    <AdditionalFiles Include="$(MSBuildThisFileDirectory)\BannedSymbols.txt" />
```

```suggestion
    <AdditionalFiles Include="$(MSBuildThisFileDirectory)BannedSymbols.txt" />
```

`MSBuildThisFileDirectory` by definition already contains a trailing `\`.